### DISCUSSION TITLE
How do I group two parent nodes into one "cloud" or into one distict group?

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1550] [Help]
<img width="605" alt="jdfidsfdsdsfa" src="https://github.com/freeplane/freeplane/assets/147225942/e3df67cc-903e-4feb-a8f2-bcb5f60fc850">

Instead of how there are two, seperate clouds for the parent nodes "include" and "don't include", I'd like to use one cloud for both series of nodes. Hopefully, this can be done!

**Comment by EdoFro** [comment]
1. Insert a parent node to "include" and "Don't include".
2. Add a cloud to it
3. hide that node (make it a hidden node).
